src/logging.rs

```diff
@@ -38,7 +38,7 @@ pub fn init_tracing() -> Result<()> {
     Ok(())
 }
 
-pub fn init_tracing_with_nats(nats: TypedNats) -> Result<()> {
+pub fn init_tracing_with_nats(nats: TypedNats, subject: String) -> Result<()> {
```

Comment:
nit: &str rather than String.

Reply:
I did a `String` here so that I could avoid lifetime issues in the `tokio::spawn` below. I could also stick `let subject = subject.to_string();` at the top of this function, too, though? Are there other ways of dealing with this?

Reply:
Ah, right. This is fine, then. LGTM.